### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by mxstbr** [mstoiber/pgp.asc#1]
Let's discuss the design! Ideas?

**Comment by iamwebrocker** [comment]
Make the logo bigger. 
scnr ;-))

**Comment by mxstbr** [comment]
To get the discussion started, I mocked up a very basic layout in balsamiq:

![first layout idea](https://cloud.githubusercontent.com/assets/7525670/5682774/c338b832-9822-11e4-9e41-528a9b5f4155.png)

**Notes**

The image placeholder are instead of sketches, similar to [humanstxt](http://humanstxt.org). I imagine the first one being several stickmen and -women holding up over-sized keys over their heads. The second one could be similar to [this one by humanstxt](http://humanstxt.org/img/Illustration-home/03-where-is-located.png), and for the third one I have no idea. 

I want to add GitHub Octocat to the logo-row at the bottom, but they don't have that.

I think we need to include a CTA saying "I have never heard of PGP" or smth similar somewhere, to get Focus Group 3 (People who have no idea what PGP is) to read the guide, which is why I added the link underneath the navbar. Any better ideas?

The [balsamiq (.bmml) file](https://mega.co.nz/#!x981ET5A!1l9zaF9NXob7iTZ1pEINXWLH-up0es_03_7teCopHFI) to play around with.

**Comment by JurgisMegnis** [comment]
Hi, guys!
My name is Jurgis and I would like to contribute to the project from design point of view. I have attached design for the landing page that I have created. Feel free to comment.
The main thing that I wanted to accomplish was to create light, simple design, so the main focus is on content. For the icons I chose to create some maze like icons. (encrypted messages really associates with maze for me, as both have some certain surroundings to prevent someone to get to the main point or main message.)

Would like to hear your thoughts on the whole 'feel' of the site. If you think it's okay, I would like to proceed on designing other sections of the site.
Cheers!

![01_pgp_landing](https://cloud.githubusercontent.com/assets/7488270/5898742/c6e4f594-a552-11e4-9502-078c779778fe.png)

**Comment by mxstbr** [comment]
I love it, especially the hexagon-maze idea and the colors. Maybe we could use the Key-Maze as a logo, I think it'd work well above the main pgp.asc title. 

We can get rid of the contribute section in the footer if we have the GitHub Link there anyway. Also I'd drop the Team Page, it should integrate with GitHub but that's too complicated to set up for now.

Other than that, I'm not a fan of the headline font, but that is easy to change later on. Great work, I sure am looking forward to the rest of the pages!

**Comment by JurgisMegnis** [comment]
Thanks for feedback!

I actually had the same thought- that we could use the Key-maze as logo. I think it'd work fine as logo.
Okay about the Team page, but I'm not sure about Contribute section, as it sits fine in the footer and it is some sort of CTA, while the GitHub link kinda blends in with other social links.

But yeah, I am glad, that you like the main feel of the design. Then I can get on the rest of the pages.

**Comment by mstoiber** [comment]
> as it sits fine in the footer and it is some sort of CTA, while the GitHub link kinda blends in with other social links.

Makes sense if you see it like that, I think you're right and we should leave it.

**Comment by JurgisMegnis** [comment]
At this point I can only share directory with .psd file, cos I have a meeting to attend in 5 minutes. 
If you can slice it yourself- fine, but it is also no problem if I have to do it. I can try to do it in the evening or, latest, tomorrow morning. 

Link - https://drive.google.com/open?id=0B9pWAAQGrgrnRjFFekNad0FqbzQ&authuser=0

**Comment by mxstbr** [comment]
> If you can slice it yourself- fine, but it is also no problem if I have to do it. I can try to do it in the evening or, latest, tomorrow morning.

That's perfectly fine, it's not like we have a deadline tomorrow morning. :wink:

**Comment by JurgisMegnis** [comment]
Here is directory with .psd file and assets - https://drive.google.com/open?id=0B9pWAAQGrgrnY25mTVdiSkluMGs&authuser=0

**Comment by mxstbr** [comment]
I think [Merriweather](http://www.google.com/fonts/specimen/Merriweather) would be a very nice headline font to use with open sans as body copy, I'll try that in the coded version.

**Comment by mstoiber** [comment]
Just coded the first part, the header is basically finished. I'm not sure how we're going to do the grid, it seems unnecessary to use an external one. Any ideas @iamwebrocker @anselmh ? (See commit 7361f7684e62a785d3ba714077fe9bd45a2512cf)

**Comment by mxstbr** [comment]
I've coded out the homepage, it's as good as finished. You find it in the [feature/design](https://github.com/mstoiber/pgp.asc/tree/feature/design) branch, please pull it and have a look. 

Everything is in SVG ([Why SVG?](http://code.tutsplus.com/articles/why-arent-you-using-svg--net-25414)), except for the submit button, but I'll wait for the hover state on that. I'm also using Merriweather, but am not happy with it quite yet. If anybody has any ideas, feel free to tell me!

The whole body markup is new, as well as all the SCSS except for normalize.

Great work so far Jurgis!

![pgp.asc website screenshot](https://cloud.githubusercontent.com/assets/7525670/5922597/d801b25c-a644-11e4-82c3-42298391e7b0.png)

**Comment by anselmh** [comment]
Looks good to me. Yet, mobile doesn’t really work, about and hall of fame page are missing. And, why do you swap out the svg sources on hover? That always means a new http request on demand via JS which isn’t too cool.

**Comment by mxstbr** [comment]
> Yet, mobile doesn’t really work

Yeah, it also looks bad on my 13" MBP because the arrow is too far down. I'll have to do the front-page differently.

> about and hall of fame page are missing

Waiting if Jurgis has anything special planned for them before I go ahead and code them out.

> And, why do you swap out the svg sources on hover? That always means a new http request on demand via JS which isn’t too cool.

That is definitely not cool, but I couldn't think of another way of doing the hover effect. Feel free to fix it if you've got a better idea!

**Comment by JurgisMegnis** [comment]
> Waiting if Jurgis has anything special planned for them before I go ahead and code them out.

Yeah, I actually have, but as I am quite busy with my dayjob at this moment, I can't spend much time on it at this point.

**Comment by iamwebrocker** [comment]
Hi guys, designer-me really like the visuals of this, but sad panda from front-end-me that we (who should know better) treat this desktop-down, not content/mobile first ;-)
sadly I'm far too engaged in dayjob work to contribute any useful stuff atm, hope this will change soonish
love the maze-logo thing!

**Comment by mstoiber** [comment]
@iamwebrocker Fixed some of the responsive problems, it looks alright on my MBP now, but it feels really hacky what I'm doing. I probably shouldn't scale everything based on height/width, do you have any better ideas?

**Comment by mxstbr** [comment]
> @anselmh And, why do you swap out the svg sources on hover?

Changed that to CSS' :hover and background-image, should be better performance-wise. (I hope) 

Also restructured the Markup for the pgpguide and the halloffame, so they now exist, but still don't look any good.

**Comment by JurgisMegnis** [comment]
> Maybe we shouldn't add new members to the bottom, but rather to the top, that way everybody can have their "Five minutes of pgp.asc fame".

Good idea. I think this could be a good solution.

> Also, how do you imagine the homepage on mobile?

I will make a quick sketch about it as soon as possible.

P.S. Sorry, that I posted my previous designs under PGP Guide, I guess I got lost in GitHub. I am gonna repost the designs here together with mobile homepage design.

**Comment by JurgisMegnis** [comment]
I am going to repost my designs here, as I posted them under PGP Guide by accident. Plus I am adding design for index page on mobile device. Looking forward to your comments. Cheers!

![02_pgp_desktop_hall](https://cloud.githubusercontent.com/assets/7488270/6097758/92dd086e-afc9-11e4-81a5-e7e98c1a1653.png)
![03_pgp_desktop_guide](https://cloud.githubusercontent.com/assets/7488270/6097759/948fe550-afc9-11e4-9f2e-ec3924419689.png)
![02_pgpasc_mobile_hall](https://cloud.githubusercontent.com/assets/7488270/6097760/963ddf60-afc9-11e4-8e9b-b7aa0e43fc77.png)
![01_pgpasc_mobile_index](https://cloud.githubusercontent.com/assets/7488270/6097761/aba8b71c-afc9-11e4-9b45-90298a6686b2.png)

**Comment by mxstbr** [comment]
Not digging the center-aligned paragraphs on the home page honestly, makes it hard to read, though I like the centered headlines much more than the left-aligned ones right now. 

I really like the rest though, if you could upload the hamburger icon to the GDrive as a .svg I'm going to finish the mobile nav today!

**Comment by JurgisMegnis** [comment]
Should be there now. 

Yeah, I have been thinking about the paragraphs as well. This way it is good because everything is aligned, but I agree that the paragraph might be harder to read.

**Comment by mxstbr** [comment]
Responsive Nav is now implemented, please have a look and tell me what you think! I had to change the markup a bit, but it was nothing drastic. 

Any ideas for the animation of it changing visibility?

**Comment by JurgisMegnis** [comment]
Good job, mate!
About animation- I thought that kinda classic - when the navigation slides out- would be cool. What do you think?

**Comment by mxstbr** [comment]
Thanks! I tried that, but it didn't look good in this case. I really like the WhatsApp Web Menu & Animation:

![menuanimation](https://cloud.githubusercontent.com/assets/7525670/6109644/36044868-b074-11e4-87f7-b8b1c705e026.gif)

Maybe we could build something similar?

**Comment by mxstbr** [comment]
Or how about a full screen navigation like this for mobile? http://codyhouse.co/demo/rounded-animated-navigation/index.html

**Comment by JurgisMegnis** [comment]
Yeah, the last one is cool. We should try to use that one and see how does it goes together with the site.

**Comment by mstoiber** [comment]
The mobile navigation is now fullscreen, but I'm still not sure that this is the right direction. Not happy with the result, and I don't want to make the effort to implement the animation until we are 100% alright with the design. Any other ideas maybe?

**Comment by mxstbr** [comment]
Alright, a first idea for the navigation is done and up. (feature/design)

I have yet to implement the animation because that is quite elaborate and I don't want to dive into that if we're not all happy with the direction this is going. Anybody have any comments, maybe some design input @JurgisMegnis? 

EDIT: Also, how do you imagine the Table of Contents of the PGP Guide to look on mobile?

**Comment by mxstbr** [comment]
After a way too long time, (sorry about the delay) the first version of the new pgp.asc website is online! Check it out: http://pgpasc.org /cc @JurgisMegnis
